+hw/+ptb/Window.m

```diff
@@ -197,9 +197,10 @@ function open(obj)
       end
       Screen('Preference', 'SuppressAllWarnings', true);
       % setup screen window
+      %obj.PxDepth = 32;
```

Comment:
Instead of commenting things out with no explanation, leave some sort of description or remove entirely.  Otherwise it's just clutter that will mean nothing to no one in a few month's time.  

[MATLAB doesn't save parameters whose defaults are unchanged](https://uk.mathworks.com/help/matlab/matlab_oop/understanding-the-save-and-load-process.html).  This means that you might end up breaking someone's rig by making this change.  If you want to go ahead with it let everyone know on slack to check this works on their rigs _before_ merging into master.

Reply:
this was a little egg for you : ) want to see if there's any reason it's set to 32? 
Regarding the not saving of this property, I tested it out on the training rigs and BII and all was good, so i think should be fine, but I can send a message to the slack as well

Reply:
I don't know enough about graphics tbh but there's often a good reason for things to be set as they are.  It may be that controlling the bit depth is essential for generating the visual stimuli correctly.  If you let the system determine the value I suppose it will vary across rigs which we don't necessarily want.  I know that bit depth is important in rendering low contrasts and that there are some assumptions made in parts of the code (e.g. https://github.com/cortex-lab/signals/blob/9f4f5d068805d8dc715e332449086e42fa396da2/%2Bvis/gaussianLayer.m#L18-L22).  From what I've read 32 is an odd value for bit depth, and it may simply be a carry-over from when we used a particular monitor.

Long story short, I've come to realize that one shouldn't flippantly change bits of code when you don't understand them.  You should ask someone like Krumin about this and make sure everyone tests their recording rigs or at least approves this change.

Reply:
yup fair point, I'll do this